Dockerfile

```diff
@@ -48,15 +48,15 @@ RUN sed -i -e 's/^nexus-context-path=\//nexus-context-path=\/\${NEXUS_CONTEXT}/g
 #   nexus:x:200:nexus
 RUN groupadd --gid 200 nexus && \
     useradd \
-      --system \
-      --shell /bin/false \
-      --comment 'Nexus Repository Manager user' \
-      --home-dir /opt/sonatype/nexus \
-      --no-create-home \
-      --no-user-group \
-      --uid 200 \
-      --gid 200 \
-      nexus
+    --system \
+    --shell /bin/false \
+    --comment 'Nexus Repository Manager user' \
+    --home-dir /opt/sonatype/nexus \
+    --no-create-home \
+    --no-user-group \
+    --uid 200 \
+    --gid 200 \
+    nexus
```

Comment:
Thanks for this - can we remove these spacing changes then I"ll merge this.

Thanks!

Reply:
I'm using Prettier and it says document is already formatted properly.